Pugzor.Core/PugRendering.cs

```diff
@@ -5,21 +5,29 @@
 using Microsoft.AspNetCore.Mvc.ModelBinding;
 using Pugzor.Core.Abstractions;
 using Pugzor.Core.Helpers;
-
+using Microsoft.Extensions.Options;
+
 namespace Pugzor.Core
 {
     public class PugRendering : IPugRendering
     {
         private readonly INodeServices _nodeServices;
+        private PugzorViewEngineOptions Options;
```

Comment:
please follow the naming format used elsewhere in the program:
`private readonly PugzorViewEngineOptions _options;`